src/App.js

```diff
@@ -4,16 +4,25 @@ import { getFishData } from './ApiCalls';
 import Fishes from './Fishes';
 import Focus from './Focus';
 import SavedFishes from './SavedFishes';
+import PageNotFound from './PageNotFound';
 import Nav from './Nav';
-import { Router, Routes, Route } from 'react-router-dom';
+import { Router, Routes, Route, Navigate } from 'react-router-dom';
 
 function App() {
   const [fishes, setFishes] = useState([]);
   const [saved, setSaved] = useState([]);
+  const [error, setError] = useState("");
 
   useEffect(() => {
     getFishData()
     .then(data => setFishes(data))
+    .catch(error => {
```

Comment:
clean code . great to see how your passing the setError(error) state through your parent compnent.